layouts/lessen/single.html

```diff
@@ -1,20 +1,15 @@
 {{ define "header" }}
-  <header>
-    {{ partial "header/site-navigation.html" . }}
-  </header>
-{{ end }}
-
-{{ define "mathjax" }} {{ partial "head/mathjax" . }} {{ end }}
-{{ define "bokeh" }} {{ partial "head/bokeh" . }} {{ end }}
-{{ define "fancybox" }} {{ partial "head/fancybox" . }} {{ end }}
-
-{{ define "json_ld" }}
-  {{ partial "head/seo/organisation" . }}
-  {{ partial "head/seo/breadcrumb" . }}
-  {{ partial "head/seo/article" . }}
-{{ end }}
-
-{{ define "main" }}
+<header>
+  {{ partial "header/site-navigation.html" . }} {{ block "meta-tags"}}
+  <meta name="description" content="{{ $.Params.description | $.Summary }}" />
```

Comment:
Hier ben je vergeten om `default` achter de pipe te zetten. Het moet dus `{{ $.Params.description | default $.Summary }}` zijn, of zonder de dollartekens `{{ .Params.description | default .Summary }}`.

Verwar die pipe zeker niet met een `or` of zo. Het is eigenlijk beter te vergelijken met een pipe in unix, zoals `ps aux | grep vim`. De uitgang van het vorige commando dient als laatste argument van het commando achter de pipe. [`default`](https://gohugo.io/functions/default/#readout) is een Hugo functie die twee argumenten heeft: `DEFAULT` en `INPUT`. Wanneer `INPUT` leeg is, returnt hij `DEFAULT`.

`{{ .Params.description | default .Summary }}` is dus hetzelfde als `{{ default .Summary .Params.description }}`

Reply:
Aha oei, goed dat je het zegt, want da's een serieus verschil met hoe dit soort dingen in andere talen gebruikt wordt. Dan heeft die code niet veel zin, want helemaal verkeerd :D Bedankt voor de feedback!